### ISSUE TITLE
Unable to proceed to the data check-up step

### ISSUE THREAD

**Issue by cecilaki** [spendb/spendb#35]
Hi! 
I'm a basic user using the interface to load and edit my data ;-)
I was proceeding to some upload but I'm sort of stuck in this step where I can't validate my data (I'm on the check-up step) and can't go further. I'd like to understand what is wrong with the data so I can fix it, or edit it, but I don't see any guidance to help me. 

Any idea? 
Many thanks!

**Comment by pudo** [comment]
Hello basic user Cecile ;) -- can you share the link to the dataset that you've uploaded?

My guess: the demo version of SpenDB at spendb-dev.herokuapp.com is running on a free plan at Heroku and may have just ran out of monthly data juice. Need to move this to my server again :(

**Comment by cecilaki** [comment]
Allright I see. Fair enough :) 

Here's the dataset > https://spendb-dev.herokuapp.com/api/3/datasets/executed-budget-paris-2015/serve/comptes-administratifs-etats-speciaux-darrondissement.csv

**Comment by pudo** [comment]
yeah, looks like the service is out of steam. damn. you can try the one at http://db.offenerhaushalt.de/ if you like, that's running on a proper server.

**Comment by cecilaki** [comment]
Sure! On it.

That said, I can suggest adding a loading icon or notification in the UI saying my dataset was uploaded, cause I didn't realized it until I got stuck in the source-data step. I've been through my dashboard, where I understood I actually uploaded several datasets in my several attempts to upload. 

Apparently there was something wrong with the original dataset I wanted to upload. But I don't know what. Which is why I've uploaded another one to test. 

But then I realized also I couldn't delete all the datasources in my dashboard. I could only add a new one.

Is that built in purpose?

**Comment by pwalsh** [comment]
@pudo are you on the free plan at the moment?

**Comment by pudo** [comment]
@pwalsh yep; so both my postgres and rabbitmq cut out at some point. SpenDB is dockerized, just need to get a demo running on a proper host :)

**Comment by pudo** [comment]
@cecilaki thanks for these points, very useful. I don't think anything is wrong with the data you uploaded (which is why you're not seeing an error), the process simply stops because of .. me not paying people money. 

Deleting sources currently isn't implemented, that's a good point to add!

**Comment by cecilaki** [comment]
@pudo Very good to know. Ok, so it's just a few human readable information to add so the process gets smoother. Yeah. I'm basic-user-oriented ;-)

**Comment by pudo** [comment]
Oh yes, plenty of that. If you manage to load it into the other instance, I'd love to hear about your experience with the model editor (dimensions, measures) -- that's the hardest part to communicate, I feel!

**Comment by cecilaki** [comment]
@pudo Sure I'll tell you everything about it. For the moment I face an error in my dataset. ( or it's maybe an acceptance criteria of your validator?) Does that mean there's an issue with the headers and it might have some empty cells somewhere in the file? 

--> Error at row 0: 'NoneType' object has no attribute 'value'
Converted 1 rows with 0 columns.

**Comment by rotecodefraktion** [comment]
@cecilaki @pudo did you solve the --> Error at row 0: 'NoneType' object has no attribute 'value' issue?
I run into the same trouble, when i want upload new dataset from csv.
